src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/ActionSvcClientImplTest.java

```diff
@@ -62,6 +63,9 @@ public void testCreateActionPlan() {
     actionPlanDTO.setName(ACTION_PLAN_NAME);
     actionPlanDTO.setDescription(ACTION_PLAN_DESCRIPTION);
     actionPlanDTO.setCreatedBy("SYSTEM");
+    HashMap<String, String> selectors = new HashMap<>();
```

Comment:
A nice util here would be `Collections.singletonMap("testSelector", "testValue")`

Reply:
Not sure if this works for `HashMap`, another reason to implement your suggestion above

Reply:
I think it can be done like this`Collections.synchronizedMap(new HashMap("testSelector", "testValue"))` feel free to correct me if I'm wrong?

Reply:
Had a try and couldn't get it working @MIslam92 
Might just be the way I implemented it but It'll do for now as is anyway :+1:

Reply:
you're unlikely to need a synchronised collection in a unit test

Reply:
What Ben said is good... saves a line of code. Again, you should prefer to 'hide' the implementation by using Map as the local variable instead of the concrete type... it's Java convention/best-practice.

Reply:
@MIslam92 no need to use a synchronised collection anywhere that's not multithreaded

Reply:
I would say no need to use a an Atomic object if you are ina Thread safe context. For example a Servlet has share context so you are not offcialy in a multithreading condition ( actually you are but it s transparent ) and is not thread safe.

Reply:
@benjefferies  not sure he needs to use a Collections.singletonMap. Is it going to be mutable or stay like this ? The main difference is not just about having the code shorted but the fact that this will create an immutable Map.